entities/communication/voiceCallSetting.json

```diff
@@ -0,0 +1,99 @@
+{
+    "type": "object",
+    "properties": {
+        "uid": {
+            "type": "string",
+            "description": "The voice call settings unique identifier."
```

Comment:
It reads strange, please make it "The VoiceCallSetting unique identifier (its also singular not plural)